lib/utils.pm

```diff
@@ -430,6 +430,8 @@ sub workaround_type_encrypted_passphrase {
     # nothing to do if the boot partition is not encrypted in FULL_LVM_ENCRYPT
     return if get_var('UNENCRYPTED_BOOT');
     return if !get_var('ENCRYPT') && !get_var('FULL_LVM_ENCRYPT');
+    # for Leap 42.3 and SLE 12 codestream the boot partition is not encrypted
+    return if (is_leap('=42.3') || is_sles('<15')) && get_var('UEFI') && get_var('ENCRYPT') && get_var('LVM');
```

Comment:
We could use `is_storageng` method here, as this is the difference. But at least s/is_sles/is_sle/.
We also can skip checking `ENCRYPT` variable, as this is included in the condition above, or we need to check `FULL_LVM_ENCRYPT` too.

Reply:
@rwx788 the is_sles is a typo :) Simplifying then the condition. However, by looking at the code, `!is_storage_ng && get_var('UEFI')` then should be sufficient instead of checking for the version specific code? or should that be additional?

Reply:
Apparently `!is_storage_ng && get_var('UEFI')` is more than enough http://phobos.suse.de/tests/1747025#step/grub_test/1